### DISCUSSION TITLE
Request for more Apple Lossless decode

### DISCUSSION THREAD

**Discussion by tinyreminder** [MusicPlayerDaemon/MPD#1370] [Ideas]
Hi,

MPD currently support only FFmpeg as Apple Lossless (ALAC) decoder.

There are a few more ALAC decoder such as:
https://github.com/mikebrady/alac
https://audiofile.68k.org/

I would like to try them. 
Could you consider about supporting them as ALAC decoder?

Takeshi

**Comment by MaxKellermann** [comment]
I'm not interested in ALAC support, but if you write a MPD plugin for these libraries, submit a PR, and I'll consider merging it.

**Comment by tinyreminder** [comment]
Thank you for your reply.

I'm trying to add ALAC and FLAC decode to audiofile plugin.
FLAC works with just adding MIME type to the array
but ALAC does not...
I am confused about MIME type of ALAC😵‍💫

I will commit if I can fix this issue.
